queries.yaml

```diff
@@ -542,6 +552,23 @@ post_process:
       source_layer: roads
       properties: [layer]
 
+  # supersede existing osm buildings with data from landmark geojson layer
+  - fn: vectordatasource.transform.overlap
+    params:
+      base_layer: buildings
+      cutting_layer: landmarks
+      attribute: superseded
+      target_attribute: superseded
+      linear: false
+
+  # turn the landmark geojson polygons into point features
```

Comment:
the final output of the landmarks layer is a point layer, basically landmark pois with a bunch of metadata about the landmark. the mutate transform lets us both modify the geom and the properties with python code that gets `eval`'d directly. pretty powerful stuff. you can reference existing `properties` of a feature using the `{properties}` tag and you can access the geometry using the `{geometry}` tag. in the future we might want to allow more stuff to be referenced.